crates/cxx-qt-build/src/lib.rs

```diff
@@ -498,6 +498,26 @@ impl CxxQtBuilder {
             qtbuild.version().major.to_string(),
         );
 
+        for minor in 0..qtbuild.version().minor {
+            let at_least_qt_version = format!(
+                "cxxqt_at_least_qt_version_{}_{}",
+                qtbuild.version().major,
+                minor
+            );
+            println!("cargo:rustc-cfg={}", at_least_qt_version);
+            let variable_cargo = format!("CARGO_CFG_{}", at_least_qt_version);
+            env::set_var(variable_cargo, "true");
+        }
+
+        // We don't support Qt < 5
+        (5..qtbuild.version().major).for_each(|_: u32| {
```

Comment:
why is this one using `for_each` whereas above uses `for`, can these be consistent ?

Reply:
I think that for_each was better but indeed I can use for _ in 5..qtbuild.version().major
as you want.